cell.hoc

```diff
@@ -69,6 +69,7 @@ proc biophys() {
     insert pas
       g_pas = 0.001
       e_pas = -65
+    insert hh
```

Comment:
@nrnhines: is this is for "demonstration purposes? i.e. to make cell artificially heavy by having an instance of hh channel in every compartment of dendrite? Asking to clarify if this should be merged into master like this.

Reply:
Yes. This might better be left out but  it's not a hill to die on.
The important part to me is the hands-on.ipynb, but turning off the extra hh will likely change the reference_data.  Olli is right about the desirability of this being in the master before the ebrains workshop. We could always revert the hh part.

Reply:
By the way. I initially added it to see if the soa performance issue remained but that has been satisfactorily resolved.

Reply:
Ok.